core/modules/menu/menu.admin.inc

```diff
@@ -418,15 +420,31 @@ function menu_edit_item($form, &$form_state, $type, $item, $menu) {
   if (!isset($options[$default])) {
     $default = 'main-menu:0';
   }
+  $form['parent_menu'] = array(
```

Comment:
Let's add some documentation here as to how this works and why it has the `js-show` class.

Reply:
Done